CRM/Usermenu/Hooks/CoreResourceList/HomeMenu.php

```diff
@@ -0,0 +1,65 @@
+<?php
```

Comment:
Please add the phplinter to this repo and fix the linting errors.

Reply:
Also add the JS and scss linters too.

Reply:
@tunbola the linters have been added in the previous PR. We rebased this one.